crypto/evp/digest.c

```diff
@@ -583,11 +609,21 @@ const OSSL_PARAM *EVP_MD_gettable_ctx_params(const EVP_MD *md)
 
 const OSSL_PARAM *EVP_MD_CTX_gettable_params(EVP_MD_CTX *ctx)
 {
+    EVP_PKEY_CTX *pctx = ctx->pctx;
+
     if (ctx != NULL
```

Comment:
Same here, the NULL check needs to go before the dereference.

Reply:
Fixed.